src/main/webapp/courseLoader.js

```diff
@@ -380,77 +380,4 @@
       createComment(comment, isCourse);
     }
   }
-
```

Comment:
In the future, when simultaneously moving a lot of code like this to a new file and then adding to it, please do this in 2 PRs. That would make it much easier for your reviewers to see exactly what additions you made and what was simply moving code between files.

Reply:
Note taken, sorry about that! :)